app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -101,6 +101,42 @@ def self.params_for_create(ems)
             }
           end,
         },
+        {
+          :component    => 'select',
+          :name         => 'affinityPVMInstance',
+          :id           => 'affinityPVMInstance',
+          :label        => _('Cloud Volume Type'),  
+          :validate     => [{:type => 'required'}],
+          :condition    => {
+            :when    => 'affinity_policy',
+            :pattern => '^Off$',
+          },
+          :includeEmpty => true,
+          :options      => ems.vms.map do |cvt|
```

Comment:
The `ems` here is actually a `StorageManager` and VMs are owned by a `CloudManager` so this won't work. You'll have to do something like this:
```ruby
          :options      => ems.parent_manager.vms.map do |vm|
            {                             
              :label => vm.name,                
              :value => vm.ems_ref,
            }
          end,
```